code/game/objects/items/binoculars.dm

```diff
@@ -198,6 +213,15 @@
 		return
 	if(user.do_actions)
 		return
+	if(mode == MODE_MORTAR)
+		if(!linked_mortar)
+			to_chat(user, "<span class='notice'>No linked mortar found.</span>")
+			return
+		targetturf = TU
+		to_chat(user, "<span class='notice'>COORDINATES TARGETED: LONGITUDE [targetturf.x]. LATITUDE [targetturf.y].</span>")
+		playsound(src, 'sound/effects/binoctarget.ogg', 35)
+		linked_mortar.recieve_target(TU,src,user)
```

Comment:
You probably want to use signals for that.
That way, no need to link anything, any mortar could listen to a global signal and fire on its coordinate

Reply:
huh, i might ask ye about that.